crates/parcel/src/request_tracker/request_tracker.rs

```diff
@@ -35,33 +35,32 @@ use super::{RunRequestContext, RunRequestMessage};
 ///
 /// This will be used to trigger cache invalidations.
 pub struct RequestTracker {
-  graph: RequestGraph<RequestResult>,
-  reporter: Arc<CompositeReporterPlugin>,
-  request_index: HashMap<u64, NodeIndex>,
   cache: CacheRef,
```

Comment:
This looks fine really but alphabetical ordering of fields is not a good **rule** in native because the order here is the order in memory and the memory layout can matter for certain things (not this though)

Reply:
Sorry, I'm a bit obsessive over it. Isn't this only relevant to non-Rust representations though?

Reply:
I wouldn't think so; though maybe without repr(C) there's no guarantee the compiler won't mess around with the layout.

Anyhow I'm just commenting, this is not a request to change or anything like that.